### ISSUE TITLE
Test grouping in reports

### ISSUE THREAD

**Issue by JakeGinnivan** [TestStack/TestStack.BDDfy#185]
It would be handy to be able to separate tests into groups. 

With specflow you can have many different .feature files which keeps everything separated. BDDfy puts all tests from a single run into one report.

We should be able to:
1) Output tests into multiple reports
2) Filter/group tests in the html report live in the browser

Thoughts?

**Comment by karlmet** [comment]
I don't think we need to filter out scenarios tests like when overriding the RunsOn on the HtmlConfig.

If we don't want a big report file, there should be an option to produce as many report files as we have test classes. Not just one that contains them all. 

That way, our business analysts will have an easier way to check up the reports. They will only open the one with the story/class they are interested in. Is that possible right now ? If not, can this be a new feature in a futur release ?

**Comment by MehdiK** [comment]
@karlmet Thanks for the comment.

I don't recommend doing that. That would result into too many files; that said you can use `RunsOn` on config to put every class into its own report if you want. It should be just a couple of lines of code. See [this](https://github.com/TestStack/TestStack.BDDfy/blob/master/Samples/TestStack.BDDfy.Samples/Atm/AtmHtmlReportConfig.cs#L9) for an example.
